modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -4388,6 +4418,7 @@ void GDScriptAnalyzer::reduce_subscript(GDScriptParser::SubscriptNode *p_subscri
 				mark_node_unsafe(p_subscript);
 			}
 		}
+
```

Comment:
Previous if(!valid) may be unified with the next statement

Reply:
No, this would change the logic since `valid` is reassigned above.

Reply:
Ah, right...